frontends/web/src/routes/account/receive/receive.tsx

```diff
@@ -144,14 +146,22 @@ class Receive extends Component<Props, State> {
         return this.props.devices[this.props.deviceIDs[0]];
     }
 
+    private toggleAddressType = () => {
+        this.setState(({ addressType }) => ({
+            // TODO: will have an aditional type once we implement Taproot
```

Comment:
imho TODO's should be only for stuff that can be worked on immediately ;)

Reply:
should I remove the comment?

Reply:
yeah i would. in general todo's are best filed in issues, otherwise the code is littered with TODOs that are often never resolved.